internal/daemon/controller/rate_limiter_test.go

```diff
@@ -63,7 +63,7 @@ func Test_newRateLimiterConfig(t *testing.T) {
 			ratelimit.DefaultLimiterMaxQuotas(),
 			false,
 			&rateLimiterConfig{
-				maxSize:  296148,
+				maxSize:  308154,
```

Comment:
We could calculate it in the test, but I hesitate to do so since it would duplicate the logic that is being tested here. We would expect this to change when new endpoints are added.